cmd/reading/list/list.go

```diff
@@ -38,11 +35,26 @@ var limit int32
 // NewCommand returns the list device command
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "list",
-		Short: "A list of all device readings",
-		Long:  `Return all device readings.`,
-		Args:  cobra.MaximumNArgs(1),
-		Run: func(cmd *cobra.Command, args []string) {
+		Use:                    "list",
```

Comment:
We do not need all the default values of cobra.Command struct

Reply:
Is there any benefit to leaving them there?  Or for omitting?

Reply:
For omitting - until we decide to change the values different than default